static/js/components/SiteContentListing.test.tsx

```diff
@@ -33,7 +34,14 @@ jest.mock("react-router-dom", () => ({
 }))
 
 // ckeditor is not working properly in tests, but we don't need to test it here so just mock it away
-jest.mock("./widgets/MarkdownEditor", () => "div")
+function mocko() {
```

Comment:
It was an issue related to the `minimal` prop we use for the MarkdownEditor. Basically, React throws an error if you pass a random unexpected prop to an HTML element (like doing `<div foobar={bizBaz} />`), and we were mocking the MarkdownEditor component with a simple `div`. So then the extra props like `setFieldValue` and `minimal` were, because of the way that Formik works, being passed through to the `div` and causing an error. So to fix it I just defined a little custom component to use as a mock instead, since a custom component can receive any prop at runtime and won't throw any error.

Does that make sense? It's sort of hard to explain :thinking: